tests/integration/targets/ecs_cluster/tasks/main.yml

```
@@ -223,6 +225,11 @@
         role: "ecsServiceRole"
       register: ecs_service
 
+    - name: check that placement constraint has been applied
+      assert:
+        that:
+          - "ecs_service.service.placementConstraints[0].type == 'distinctInstance'"
+
```

```suggestion
    - name: check that placement constraint has been applied
      assert:
        that:
          - ecs_service.changed
          - "ecs_service.service.placementConstraints[0].type == 'distinctInstance'"

```

@novak-as  you can merge this with the assert in line 233  
otherwise LGTM